holidays/countries/saint_helena_ascension_and_tristan_da_cunha.py

```diff
@@ -0,0 +1,196 @@
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Saint_Helena,_Ascension_and_Tristan_da_Cunha>
+        * [Ratting Day](https://web.archive.org/web/20250826151056/https://www.tristandc.com/newsratting.php)
+    Saint Helena:
+        * <https://web.archive.org/web/20250810194916/https://sainthelenaisland.info/holidays.htm>
+        * [Public Holidays Ordinance](https://web.archive.org/web/20231129081100/https://www.sainthelena.gov.sh/wp-content/uploads/2013/01/Public-Holidays-Ordinance.pdf)
+        * [2015](https://web.archive.org/web/20241219152350/https://www.sainthelena.gov.sh/wp-content/uploads/2012/08/Public-Holidays-2015.pdf)
+        * [2020](https://web.archive.org/web/20250831203105/https://www.sainthelena.gov.sh/wp-content/uploads/2019/11/Public-Holidays-2020.pdf)
+        * [2021](https://web.archive.org/web/20240626101426/https://www.sainthelena.gov.sh/wp-content/uploads/2020/08/200824_Public-Holidays-2021.pdf)
+        * [2022](https://web.archive.org/web/20230922215102/https://www.sainthelena.gov.sh/wp-content/uploads/2021/11/PUBLIC-HOLIDAYS-2022.pdf)
+        * [Saint Helena Day 2023](https://web.archive.org/web/20250519085755/https://www.sainthelena.gov.sh/wp-content/uploads/2022/09/EX-GAZ-100-Proclamation-St-Helena-Day-2023.pdf)
+        * [2024](https://web.archive.org/web/20250614201000/https://www.sainthelena.gov.sh/2023/news/public-and-government-holidays-2024/)
+        * [2025](https://web.archive.org/web/20241005191212/https://www.sainthelena.gov.sh/wp-content/uploads/2024/08/Public-Notice-Public-Holidays-2025.pdf)
+        * [2026](https://web.archive.org/web/20250831200128/https://www.sainthelena.gov.sh/wp-content/uploads/2025/08/Public-Notice-Public-Holidays-2026.pdf)
+    Ascension:
+        * [Public Holidays Ordinance](https://web.archive.org/web/20240610184146/https://www.sainthelena.gov.sh/wp-content/uploads/2017/12/Public-Holidays-Asc-Ordinance.pdf)
+    """
+
+    country = "SH"
+    default_language = "en_GB"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    # Earliest year of holidays with an accessible online record.
+    start_year = 2015
+    subdivisions = (
+        "AC",  # Ascension.
+        "HL",  # Saint Helena.
+        "TA",  # Tristan da Cunha.
+    )
+    subdivisions_aliases = {
+        "Ascension": "AC",
+        "Saint Helena": "HL",
+        "Tristan da Cunha": "TA",
+    }
+    supported_categories = (GOVERNMENT, PUBLIC)
+    supported_languages = ("en_GB", "en_US")
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, cls=SaintHelenaAscensionAndTristanDaCunhaStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
```

Comment:
I meant that it's *only* a GOVERNMENT holiday officially (that's why it's strange).